### PR TITLE
doc: gsg: Windows: Clarify usage of terminal/cmd.exe

### PR DESCRIPTION
**Pull request by carlescufi**
Responding to feedback from the community, clarify that the cmd.exe window needs to be closed and reopened.

Also be specific about the type of terminal window to be used.

See https://discord.com/channels/720317445772017664/1037304376882106439/1171807232753684510

### CODE REVIEW

File: doc/develop/getting_started/index.rst [hunk 1]
```diff
@@ -148,8 +149,8 @@ The current minimum required version for the main dependencies are:
 
       #. `Install chocolatey`_.
 
-      #. Open a ``cmd.exe`` window as **Administrator**. To do so, press the Windows key,
-         type "cmd.exe", right-click the result, and choose :guilabel:`Run as
+      #. Open a ``cmd.exe`` terminal window as **Administrator**. To do so, press the Windows key,
+         type ``cmd.exe``, right-click the "Command Prompt" search result, and choose :guilabel:`Run as
          Administrator`.
```
Comment (kartben) [inline doc/develop/getting_started/index.rst:154]:
```suggestion
         type ``cmd.exe``, right-click the :guilabel:`Command Prompt`` search result, and choose 
         :guilabel:`Run as Administrator`.
```

File: doc/develop/getting_started/index.rst [hunk 2]
```diff
@@ -148,8 +149,8 @@ The current minimum required version for the main dependencies are:
 
       #. `Install chocolatey`_.
 
-      #. Open a ``cmd.exe`` window as **Administrator**. To do so, press the Windows key,
-         type "cmd.exe", right-click the result, and choose :guilabel:`Run as
+      #. Open a ``cmd.exe`` terminal window as **Administrator**. To do so, press the Windows key,
+         type ``cmd.exe``, right-click the "Command Prompt" search result, and choose :guilabel:`Run as
```
Comment (kartben) [inline doc/develop/getting_started/index.rst:153]: so is it just me or following these steps actually ends up opening the old cmd.exe terminal window? It does open windows terminal though if I launch the "Command Prompt" search result as a regular user.

**Review by kartben — CHANGES_REQUESTED** [review]
thanks! one minor cosmetic request for change and a question/comment

File: doc/develop/getting_started/index.rst (hunk 2 above)
Comment (carlescufi) [reply doc/develop/getting_started/index.rst:153]: I think it is just you, on Windows 11:

image

which is the Windows Terminal.

Comment (carlescufi) [reply doc/develop/getting_started/index.rst]: Maybe here:
image
See also https://www.makeuseof.com/make-windows-terminal-default-app/

**Review by fabiobaltieri — APPROVED** [review]

File: doc/develop/getting_started/index.rst (hunk 2 above)
Comment (carlescufi) [reply doc/develop/getting_started/index.rst]: @kartben can you take another look?

**Review by kartben — APPROVED** [review]

File: doc/develop/getting_started/index.rst (hunk 2 above)
Comment (dleach02) [reply doc/develop/getting_started/index.rst]: Hi... friend of mine who is technical was following the instructions and the first part of this about installing windows terminal sent him off into the weeds. He installed some windows terminal application and then went from there. 

I was also confused by this instruction. Why do we call it the windows terminal application when it is called the windows command prompt? Or are we trying to get someone to install something else?

**Pull request merged**